### PR TITLE
pkill and friends are more portable inexplicably

### PR DESCRIPTION
**Pull request by despair86**

### CODE REVIEW

**Comment by despair86** [comment]
according to the linux man page
```
pkill and pgrep were introduced in Sun's Solaris 7. This implementation is fully compatible. 
```

File: src/process/LinuxLokinetProcessManager.cpp [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #include <QProcess>
 #include <QDebug>
 
-#ifdef Q_OS_LINUX
+#if !defined(Q_OS_WIN) && !defined(Q_OS_MACOS)
```
Comment (notlesh) [inline src/process/LinuxLokinetProcessManager.cpp:7]: Are there other platforms that you tested this on where this works?

File: src/process/LinuxLokinetProcessManager.cpp [hunk 2]
```diff
@@ -64,7 +64,7 @@ bool LinuxLokinetProcessManager::doGetProcessPid(int& pid)
     int tmp = output.toInt(&success);
     if (!success)
     {
-        qDebug() << "Unrecognized 'pidof' output: " << output;
+        qDebug() << "Unrecognized 'grep' output: " << output;
```
Comment (notlesh) [inline src/process/LinuxLokinetProcessManager.cpp]: s/grep/pgrep/ ?

File: src/process/LokinetProcessManager.cpp [hunk 3]
```diff
@@ -113,18 +113,19 @@ void LokinetProcessManager::setLastKnownStatus(ProcessStatus status)
...
 WindowsLokinetProcessManager g_processManager;
 
 #elif defined(Q_OS_MACOS)
 #include "MacOSLokinetProcessManager.hpp"
 MacOSLokinetProcessManager g_processManager;
 
+#else
+#include "LinuxLokinetProcessManager.hpp"
+LinuxLokinetProcessManager g_processManager;
+
```
Comment (notlesh) [inline src/process/LokinetProcessManager.cpp:131]: I'm happy to support whatever platforms we get for free here -- should we add a comment about what we know works?

**Review by notlesh — CHANGES_REQUESTED** [review]
Looks good, couple minor things because I'm pedantic :)

File: src/process/LinuxLokinetProcessManager.cpp (hunk 2 above)
Comment (notlesh) [reply src/process/LinuxLokinetProcessManager.cpp]: nevermind, I see that you addressed that in your next commit

File: src/process/LinuxLokinetProcessManager.cpp (hunk 1 above)
Comment (despair86) [reply src/process/LinuxLokinetProcessManager.cpp:7]: netbsd (openbsd), freebsd

File: src/process/LokinetProcessManager.cpp (hunk 3 above)
Comment (despair86) [reply src/process/LokinetProcessManager.cpp:131]: done!

**Pull request merged**